load-testing/tests/relays_stress_test.go

```diff
@@ -57,6 +57,11 @@ const (
 	maxAmountColIdx
 )
 
+// Signing a transaction with online mode queries the node for the account sequence
```

Comment:
Can you #PUC (what we discussed) with potential links to cosmos docs on:
- What's a sequence number & account number
- Why are we using online mode?
- Explain that 3 was empirically / arbitrarily selected